Documentation/install/upgrade.rst

```diff
@@ -320,6 +320,55 @@ IMPORTANT: Changes required before upgrading to 1.8.0
   .. note:: The ENI IPAM mode automatically derives the native routing CIDR so
             no action is required.
 
+Upgrading from >=1.7.0 to 1.8.y
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Since Cilium 1.5, the conntrack table size parameter
+   ``bpf-ct-global-tcp-max`` was set to the default value ``1000000`` to retain
+   backwards compatibility with previous versions. In Cilium 1.8 the default
+   value is set to 512K by default in order to reduce the agent memory
+   consumption.
+
+   If Cilium was deployed using Helm, the new default value of 512K was already
+   effective in Cilium 1.7, unless it was manually configured to a different
```

Comment:
I think that the below should be accurate:
```suggestion
   effective in Cilium 1.6 or later, unless it was manually configured to a different
```

https://github.com/cilium/cilium/blob/215b0a49800e912d3cdeb70eb18e5bd42d6734fc/install/kubernetes/cilium/values.yaml#L191